bigquery/google/cloud/bigquery/dataset.py

```diff
@@ -127,6 +127,15 @@ def dataset_id(self):
         """
         return self._dataset_id
 
+    @property
+    def path(self):
```

Comment:
Huh, this method already exists on the branch...

Reply:
Ah, I see this appears lower down at line 180. https://github.com/GoogleCloudPlatform/google-cloud-python/pull/3973/files#diff-bafa3d472438f0a46853cc9879265cadR188

This is a duplicate definition and should be removed.

Reply:
One of the definitions is for DatasetReference, and one of them is for Dataset. We added the path method to DatasetReference because client.get_dataset() uses it.

Reply:
Oops, you're right. LGTM.